test/dlc_tests/ec2/pytorch/training/test_pytorch_training.py

```diff
@@ -181,6 +180,7 @@ def test_pytorch_linear_regression_cpu(pytorch_training, ec2_connection, cpu_onl
 @pytest.mark.usefixtures("sagemaker")
 @pytest.mark.integration("dgl")
 @pytest.mark.model("gcn")
+@pytest.mark.skip_dgl_test
```

Comment:
If a fixture is designated as `autouse`, it doesn't need to be added as as marker on the pytest test function. It should normally be automatically invoked on all test functions. Does it not work that way here?

Reply:
I see other markers like skip_inductor_test also doing same, so added